webserver/src/test/java/io/kestra/webserver/controllers/api/FlowControllerTest.java

```diff
@@ -740,6 +740,96 @@ void validateFlows() throws IOException {
         assertThat(body.get(1).getConstraints(), containsString("Invalid type: io.kestra.plugin.core.debug.UnknownTask"));
     }
 
+    @Test
+    void validateTask() throws IOException {
+        URL resource = TestsUtils.class.getClassLoader().getResource("tasks/validTask.json");
+
+        String task = Files.readString(Path.of(Objects.requireNonNull(resource).getPath()), Charset.defaultCharset());
+
+        HttpResponse<List<ValidateConstraintViolation>> response = client.toBlocking().exchange(POST("/api/v1/flows/validate/task", task).contentType(MediaType.APPLICATION_JSON), Argument.listOf(ValidateConstraintViolation.class));
```

Comment:
Possible to define the `static final String VALIDATE_TASK_URL="/api/v1/flows/validate/task"` ?